### ISSUE TITLE
"Could not find matching constructor for DefaultAndroidSourceDirectorySet" with Android Gradle plugin 7.2.0-rc01

### ISSUE THREAD

**Issue by mhsmith** [chaquo/chaquopy#613]
#### Chaquopy version
11.0.0

#### Relevant parts of your build.gradle file
```
    dependencies {
        classpath 'com.android.tools.build:gradle:7.2.0-rc01'
        classpath "com.chaquo.python:gradle:11.0.0"
    }
```
#### Describe your issue
An exception occurred applying plugin request [id: 'com.chaquo.python']
\> Failed to apply plugin 'com.chaquo.python'.
   \> Could not find matching constructor for: com.android.build.gradle.internal.api.DefaultAndroidSourceDirectorySet(String, org.gradle.api.internal.project.DefaultProject_Decorated, com.android.build.gradle.internal.api.artifact.SourceArtifactType$JAVA_SOURCES)

**Comment by mhsmith** [comment]
Chaquopy doesn't support pre-release versions of the Android Gradle plugin, so if you get this error, please go back to the current stable version, 7.1.3.

If this is still an issue when version 7.2.0 becomes stable, we'll fix it in a new Chaquopy release shortly afterwards.

**Comment by mhsmith** [comment]
This issue is fixed in Chaquopy 12.0.0. To upgrade, change the Chaquopy version number in your app's top-level build.gradle file.
